unitTests/ui/SimpleFilterTest.ts

```diff
@@ -85,6 +85,22 @@ export function SimpleFilterTest() {
       );
     });
 
+    it('should log an analytics event when a checkbox is unselected', () => {
+      aSimpleFilter.cmp.toggleValue('foo');
+      (aSimpleFilter.env.usageAnalytics.logSearchEvent as jasmine.Spy).calls.reset();
+      aSimpleFilter.cmp.toggleValue('foo');
+      const eventName = 'deselectValue';
```

Comment:
Since this variable is only referenced in one place, I think it is better to just write the string on line 96.